src/components/views/KbartFields.js

```diff
@@ -6,14 +6,16 @@ import { Field } from 'react-final-form';
 import { requiredValidator } from '@folio/stripes-erm-components';
 
 import {
+  Checkbox,
   Col,
   MessageBanner,
   Row,
-  TextField,
+  TextField
 } from '@folio/stripes/components';
 
 export default class KbartFields extends React.Component {
   render() {
+    const { localKB } = this.props || {};
```

Comment:
props is always defined, you never need to check for existence.

Reply:
This was supposed to be saying if localKB didn't exist in props then use `{}`, but I think I'm making sure that happens higher in the component tree, so I'll remove this :)

Reply:
Yeah so that line doesn't do that, actually. It evaluates `this.props` and if that is falsey, it evaluates the RHS of the `=` to be the empty object. Then, it destructures `localKB` from the empty object.

If you want a default value for a destructured variable, use the syntax:
```
const {
  thisVarShouldNeverBeUndefined = {},
  norShouldThisOne = [],
  andDefinitelyNotThisOne = 'foobar',
} = this.props;